### PR TITLE
Add new event (Lauren singing in the wilderness)

### PR DESCRIPTION
**Pull request by joellabes**

### CODE REVIEW

**Review by izzymiller — APPROVED** [review]
This may satisfy the inside joke requirement as well, since I don't get it :) LGTM!

**Comment by joellabes** [comment]
I can't find an example! but if you didn't get Lauren's recap emails last year, you are in for a TREAT.

On a more important note, I can't merge this
image

**Comment by izzymiller** [comment]
oh, humm. I'm not sure if I can turn on branch protections but also give everyone the magic merge button... I'll futz with this tomorrow!

**Pull request merged**
